sky/skylet/providers/gcp/node_provider.py

```diff
@@ -191,11 +193,11 @@ def create_node(self, base_config: dict, tags: dict, count: int) -> None:
                     filters[TAG_RAY_USER_NODE_TYPE] = labels[TAG_RAY_USER_NODE_TYPE]
                 # SKY: "TERMINATED" for compute VM, "STOPPED" for TPU VM
                 if isinstance(resource, GCPCompute):
-                    STOPPED_STATUS = ["TERMINATED"]
+                    TO_BE_STARTED_STATUS = ["TERMINATED", "STOPPING"]
                 else:
-                    STOPPED_STATUS = ["STOPPED"]
+                    TO_BE_STARTED_STATUS = ["STOPPED", "STOPPING"]
                 reuse_nodes = resource._list_instances(
-                    filters, STOPPED_STATUS)[:count]
+                    filters, TO_BE_STARTED_STATUS)[:count]
```

Comment:
The new name suggests this is going to be started, but not necessarily.  I think the old name STOPPED_STATUS is more accurate.

Reply:
Fixed!